src/app/components/CodeDisplay/index.tsx

```diff
@@ -116,6 +179,8 @@ export const JsonCodeDisplay: FC<JsonCodeDisplayProps> = ({ data, label, floatin
       copyToClipboardValue={formattedJson}
       label={label}
       floatingCopyButton={floatingCopyButton}
+      useMonaco
+      monacoLanguage="json"
```

Comment:
This also changes https://pr-1979.oasis-explorer.pages.dev/mainnet/sapphire/tx/a3fd0f84bc69abe8d2485881d57ff66c0e1e949215076c5b9bfc6dd216bdf65d

Before | After
![image](https://github.com/user-attachments/assets/815b6d3a-c87b-4b0c-882e-2a39a5c1bdc3)